docs/getting_started.md

```diff
@@ -0,0 +1,62 @@
+# TensorFlow Encrypted
+
+This guide assumes that you have followed the installation instructions in [README](https://github.com/mortendahl/tf-encrypted).
+
+- [Introduction to Tensorflow Encrypted API](#introduction-to-tensorflow-encrypted-api)
+- [Making private predictions](#make-private-prediction)
+  - [Locally](#locally)
+  - [With GCP](#with-gcp)
+- [Making private trainings](#make-private-training)
+
+## Introduction to TensorFlow Encrypted API
+
+tf-encrypted has a simple api to make it easy for data scientists to make private predictions and trainings.
+
+
+To define a machine learning model, tf-encrypted and TensorFlow follow a very similar API.
+
+TensorFlow             |  TensorFlow Encrypted
+:-------------------------:|:-------------------------:
+<img src="images/tf_layers.png" width="400" height="120" />  |  <img src="images/tfe_layers.png" width="400" height="120" />
+
+To make private predictions and trainings, we have to define a ModelTrainer and a PredictionClient. The ModelTrainer is responsible for training the model, and then provides the encrypted weights. The PredictionClient will provide the private input that will be used to make a private prediction.
+
+<img src="images/modelTrainer_PredictionClient.png" width="700" height="70" />
+
+We indicate tf-encrypted to perform secure computation with the Pond protocol as follows:
+<img src="./images/pond.png" width="600" height="23" />
+
+Finally we can make a prediction as follows:
+<img src="./images/model.png" width="720" height="400" />
+Two servers and a crypto producer are doing the actual computation on encrypted data, with only the client being able to decrypt the final result.
```

Comment:
This doesn't seem to be wrapping properly when you view the README, perhaps add a new line here?